CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## [Unreleased][unreleased]
 
+### Added
+
+- Enable Search in the UI, add a search box in the header and search results
+  view
```

Comment:
Can we just outline what we can serach for so it is recorded? I think it is:

- school/academy name
- school/academy URN
- incoming trust by UKPRN
- outgoing trust by UKPRN